.babelrc

```diff
@@ -1,38 +1,5 @@
 {
   "presets": [
-    [
-      "latest",
-      {
-        "loose": true,
-        "modules": false
-      }
-    ],
-    "stage-0",
-    "stage-3",
-    "react"
-  ],
-  "ignore": "app.static.js",
-  "plugins": [
-    "react-hot-loader/babel",
-    "transform-class-properties",
-    [
-      "transform-runtime",
-      {
-        "helpers": false,
-        "polyfill": false,
-        "regenerator": true,
-        "moduleName": "babel-runtime"
-      }
-    ]
-  ],
-  "env": {
-    "test": {
-      "presets": [
-        ["latest", {"modules": true}], 
-        "stage-0",
-        "stage-3",
-        "react"
-      ]
-    }
-  }
-}
+    ["./.babelrc.js", { "isNode": true }]
+  ]
+}
```

Comment:
World's smallest nitpick: You're missing the final newline on this file.

Reply:
Ah, well spotted.